src/chainlit/frontend/src/components/element/audio.tsx

```diff
@@ -0,0 +1,14 @@
+import { IAudioElement } from 'state/element';
+
+interface Props {
+  element: IAudioElement;
+}
+
+export default function AudioElement({ element }: Props) {
+  if (!element.url && !element.content) {
+    return null;
+  }
+  const className = `${element.display}-audio`;
+  const src = element.url || URL.createObjectURL(new Blob([element.content!]));
+  return <audio controls src={src} className={className}></audio>;
```

Comment:
Is there a way to display the name of the audio file in the native audio component? Or should we add it ourselves?

Reply:
Good point! Here is how it looks:

<img width="485" alt="image" src="https://github.com/Chainlit/chainlit/assets/494686/950e53b0-4ebf-4efe-94d7-ec190096e76e">

I took some inspiration from this part of the figma designs:

<img width="938" alt="image" src="https://github.com/Chainlit/chainlit/assets/494686/4a8f8336-e51d-4b30-b07f-070b60577ea2">

Reply:
nice!